test/resources/layout-shared.css

```diff
@@ -10,3 +10,10 @@ body {
     background: beige;
     font-size: 14px;
 }
+
+.test-pg__checkbox {
+    width: 15px;
+    height: 15px;
+    border: 1px solid magenta;
+    border-radius: 2px;
+}
```

Comment:
Looks like we can just get rid of the checkbox examples as they have no impact on the list group styles.